src/createServer.js

```diff
@@ -1,11 +1,17 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const { router: userRouter } = require('./routes/user.route');
+const { router: expenseRouter } = require('./routes/expense.route');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  app.use('/users', express.json(), userRouter);
+
+  app.use('/expenses', express.json(), expenseRouter);
```

Comment:
you can do it like that
```suggestion

  server.use(express.json());

  app.use('/users', userRouter);
  app.use('/expenses', expenseRouter);
```